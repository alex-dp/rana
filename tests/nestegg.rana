	nest(){
		egg(age="2d"){}
		bird(){}
	}
