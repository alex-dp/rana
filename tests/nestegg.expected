	<nest>
		<egg age="2d"></egg>
		<bird></bird>
	</nest>
